### PR TITLE
Add README

### PR DESCRIPTION
**Pull request by bluele**
Signed-off-by: Jun Kimura <jun.kimura@datachain.jp>

### CODE REVIEW

**Comment by bluele** [comment]
@3100 can you review this?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+"YUI" is japanese word to represent knot, join and connect
+
+# Short Description
+
+YUI is a lab to achieve interoperability between multiple heterogeneous ledgers. YUI provides modules and middleware for cross-chain communication as well as modules and tools for cross-chain application development, including an explorer to track status and events for cross-chain environments.
+
+For cross-chain communication, the design of YUI is based on Inter Blockchain Communication (IBC) protocol by Cosmos project, with extensions to support various Hyperledger projects.
+
+Modules for cross-chain application development includes one that implements a protocol for atomic operations between ledgers, such as atomic swap of tokens.
```
Comment (3100) [inline README.md:11]: > includes

include

File: README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+
+# Scope of Lab
+
+## Design Principles
+
+### Cross-authentication
+
+By relying on the on-chain verification of the other chain’s state, cross-chain communication can be achieved without any safety concern due to the behavior of off-chain actors
```
Comment (3100) [inline README.md:20]: > actors

actors.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Current status and future plan
+
+We have already developed modules for Hyperledger Fabric, Hyperledger Besu, and Corda. We plan to apply these modules for practical applications as well as to expand support for other blockchains.
+- At first, we are focusing on permissioned blockchain environment
+
+Aim to enable developers in Hyperledger community to introduce interoperability in their systems and develop cross-chain applications without too many changes and increase the complexity
+
+## Components
+
+Client (on-chain) module and Relayer to support various ledgers in align with the design principles
```
Comment (3100) [inline README.md:61]: > in align with

in alignment with ?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+### Support arbitrary data transfer and computation
+
+Developers can design and implement the specific protocol for its cross-chain application, not limited to token transfer use case.
+
+### Avoid adding components that introduce additional trust (apart from individual blockchains)
+
+Introducing such components may decrease the level of security of a system as it would be bounded by the lowest component in the system.
+
+Also, introducing new (off-chain) components with high fault tolerance will also lead to increased operational costs
```
Comment (3100) [inline README.md:38]: > costs

costs.

**Review by 3100 — APPROVED** [review]
@bluele 
Please check it.

**Pull request merged**